posthog/settings.py

```diff
@@ -100,7 +100,19 @@ def get_env(key):
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'whitenoise.middleware.WhiteNoiseMiddleware'
 ]
-INTERNAL_IPS = ['127.0.0.1']
+
+# Load debug_toolbar if we can (DEBUG and Dev modes)
+try:
+    import debug_toolbar # type: ignore
```

Comment:
Might be easier to just add the module to `mypy.ini`

Reply:
good idea